### PR TITLE
SPSH-713: Test Coverage fixen

### PR DESCRIPTION
**Pull request by DPDS93CT**
# Description
Change test coverage provider from v8 to babel, adjust tests.

## Approval for review
- [ ] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

### CODE REVIEW

File: jest.config.cjs [hunk 1]
```diff
@@ -44,10 +44,10 @@ module.exports = {
...
     coverageThreshold: {
         global: {
-            statements: -100, // ~99%
-            branches: -50, // ~96%
-            functions: -5, // ~99%
-            lines: -100, // ~99%
+            statements: 100, // ~99%
+            branches: 100, // ~96%
+            functions: 100, // ~99%
+            lines: 100, // ~99%
```
Comment (marode-cap) [inline jest.config.cjs:50]: Remove the old comments

**Review by kristoff-kiefer — CHANGES_REQUESTED** [review]
Bitte die Coverage-Anforderungen wieder hochziehen. So wie ich das sehe haben wir jetzt wieder 100%?

File: src/modules/person/api/personen-query.param.ts [hunk 2]
```diff
@@ -33,7 +33,7 @@ export class PersonenQueryParams extends PagedQueryParams {
     })
     public readonly vorname?: string;
 
-    @AutoMap(() => String)
```
Comment (marode-cap) [inline src/modules/person/api/personen-query.param.ts:36]: Could remove all `@AutoMap` from this file, PersonenQueryParams is no longer used in any mappings

File: src/modules/person/api/update-person.body.params.ts [hunk 3]
```diff
@@ -45,7 +45,7 @@ export class UpdatePersonBodyParams {
     @ApiProperty({ required: false })
     public readonly lokalisierung?: string;
 
-    @AutoMap(() => String)
```
Comment (marode-cap) [inline src/modules/person/api/update-person.body.params.ts:48]: Same as PersonenQueryParams, not used by any auto-mappers

**Review by marode-cap — CHANGES_REQUESTED** [review]

**Review by marode-cap — APPROVED** [review]

**Review by kristoff-kiefer — APPROVED** [review]

**Pull request merged**